content/millennium/bulk-data.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Bulk Data Access
+layout: api
+---
+
+# Bulk Data Access Overview
+
+The Cerner Ignite APIs for Millennium with Bulk Data Access provides system applications the ability to access large volumes of core clinical data for a group of individuals in alignment with standards cited in 21st Century Cures Act regulations. 
+
+FHIR Bulk data requests are executed against an organization's database asynchronously. When implementing an interface between a Cerner Millennium domain and a bulk data app, it is important that the queries inherent in the bulk data request capture only the data necessary for the app.
```

Comment:
nit: we should probably spell out 'app' -> 'application'.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/a71afee9ab4a5b1413077bb3157142ff44e2eff7#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R10

Reply:
@ebbp123  I think there were 2 instances of 'app' in that sentence and you updated one but not the other.

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR133